### DISCUSSION TITLE
现在题库似乎是上午更新一次，下午又更新一次。

### DISCUSSION THREAD

**Discussion by w5d1xixi** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#18] [General]
14号的题库，刚才做了，前两题是对的，后面的all error

**Comment by w5d1xixi** [comment]
**可以不可以理解，更新的题库后，立马去做效果比较好点？那今天的题库在6点之前可以等得到吗？XDD**

**Reply by flymeqy** [reply]
可能等不到

**Comment by MlgmXyysd** [comment]
通常都是小改动，照着前一天的题库基本都能搜到

**Comment by MlgmXyysd** [comment]
不过你要是对着id搜那估计寄，拿14和15号对比，小米把一些题目的id刷新了，还有一些题目选项的id互换了
